spec/support/flow_helpers.rb

```diff
@@ -37,6 +37,15 @@ def fill_in_applicant_screen(choices = {})
   click_on "Save and continue"
 end
 
+def fill_in_applicant_screen_no_employment_question(choices = {})
```

Comment:
What we've done elsewhere in this file is to check if a field exists before trying to fill it in. I.e. instead of adding this new method, update fill_in_applicant_screen to say

```
hoose choices.fetch(:employed, "Unemployed"), name: "applicant_form[employment_status]" if page.text.include?("Whatever the employment question is")
```